experimenter/query.py

```diff
@@ -0,0 +1,147 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, Q
+from experimenter.utils import get_problem_path, get_dataset_doc_path, get_pipelines_from_d3m
+from d3m.runtime import _get_data_and_scoring_params_from_pipeline_run as _data_score_params
+
+
+HOST = 'https://metalearning.datadrivendiscovery.org/es'
+CONNECTION = Elasticsearch(hosts=[HOST], timeout=300)
+
+         
+def get_search_query(arguments: dict = None, connection = CONNECTION, index='pipelines'):
+    index_search = Search(using=CONNECTION, index=index)
```

Comment:
You are not using the input connection object. Either make it an arg and use it, or don't make it an arg.